build-logic/uber-plugins/src/main/kotlin/gradlebuild.distribution-module.gradle.kts

```diff
@@ -1,3 +1,14 @@
+import gradle.kotlin.dsl.accessors._23cdd86de02729e5f5eded3732f08da5.kotlin
```

Comment:
🤔  Hmm, is it normal to have such generated accessors here?

Reply:
They are gone now.